### PR TITLE
Basic implementation for XGBoost

### PR DESCRIPTION
**Pull request by JeffersonH44**
@targos

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,9 +1,3 @@
...
-
-const defaultOptions = {};
-
-function xgboost(options) {
-    options = Object.assign({}, defaultOptions, options);
-}
-
-module.exports = xgboost;
+import loadXGBoost from './loadXGBoost';
+const xgboostWASM = require('../dist/wasm/xgboost');
```
Comment (maasencioh) [inline src/index.js]: seems strange to me have an import and a require in the same file, there's no way to do the require using the import notation?

Comment (targos) [reply src/index.js]: Oh and btw, please .gitignore the dist directory. Files are huge ^^.

Comment (JeffersonH44) [reply src/index.js]: the problem with the dist is that is needed for the test...

Comment (targos) [reply src/index.js]: let me try something...

**Pull request merged**